pkg/server/apis/v1alpha1/descriptors/webhook.go

```diff
@@ -26,9 +26,13 @@ var webhook = []definition.Descriptor{
 						Description: "tenant",
 					},
 					{
-						Source:      definition.Path,
-						Name:        "integration",
-						Description: "Integration name",
+						Source:      definition.Query,
+						Name:        "eventType",
+						Description: "the webhook eventType, support SCM for now",
```

Comment:
I'd prefer `sourceType`, for a given event source, there may be different event types.